bin/cocos.py

```diff
@@ -393,6 +393,10 @@ def parse_args(self, argv):
         parser.add_argument("-p", "--platform",
                             dest="platform",
                             help="select a platform (%s)" % ', '.join(platform_list))
+        parser.add_argument("--proj-dir",
```

Comment:
`cocos run` depends `cocos compile`. The parameters of `cocos run` will be also passed to `cocos compile`.
Because `--proj-dir` is used by `cocos compile`, so it should be added into `project_compile.py`.